force-app/main/default/classes/AffiliationMappingsMapper.cls

```diff
@@ -107,6 +107,56 @@ public virtual with sharing class AffiliationMappingsMapper {
         update afflSettings;
     }
 
+    /*****************************************************************************
+     * @description Updates the auto enrollment mapping on affiliation mapping record
+     * @param mappingName
+     * @param autoProgramEnrollmentStatus
+     * @param autoProgramEnrollmentRole
+     *****************************************************************************/
+    public void updateAutoEnrollmentMappings(
```

Comment:
Consider making a method that takes a list of mappings to update and performs the conversion from models to mappings, then updates all mappings in that list.